### ISSUE TITLE
Update for reprex 1.0+

### ISSUE THREAD

**Issue by jonthegeek** [yonicd/slackreprex#5]
reprex has built-in Slack style. While it isn't particularly fancy, it works for the purpose that slackreprex was meant to serve.

Update the package to "simply" combine `reprex_reprex_slack()` with `slackposts::chat_message()`. While doing so, implement mocks in tests.

I've started this update, but I want to focus on other things for the moment, so I'm logging it in a ticket (and an in-progress PR in a moment).

**Comment by yonicd** [comment]
not sure i like the solution reprex devs went with dumping output in md format for slack. slackreprex tried to give a more natural slack feel for the output

**Comment by jonthegeek** [comment]
Fair enough. In any case there's a deprecated argument that we should deal with, and we MIGHT want to use more of what they have. I could imagine, for example, using their solution for simple cases (I'm not sure we need more than md for a lot of things; Slack is gonna Slackify it either way), but then doing things like attaching sessioninfo separate from the main text of the message.
